app/views/cases/data_requests/show.html.slim

```diff
@@ -29,7 +29,8 @@ span.visually-hidden
       p.data_request_request_type
         = t("helpers.label.data_request.request_type.#{@data_request.request_type}")
         - if @data_request.request_type_note?
-          = ":<br>#{@data_request.request_type_note}"
+          =":"
+          <div>"#{@data_request.request_type_note}"
```

Comment:
Try to use slim syntax in a slim file
```suggestion
          div= "#{@data_request.request_type_note}"
```

Reply:
I've used div without the = due to using adding an id to the div if I used = it would print the id